bench/index.html

```diff
@@ -25,6 +30,13 @@
 </head>
 
 <body class="prose">
+    <div id="benchmarks">
+        <a href='/bench/buffer' class='button'>Buffer</a>
+        <a href='/bench/fps' class='button'>FPS</a>
+        <a href='/bench/query-box' class='button'>Query Box</a>
+        <a href='/bench/query-point' class='button'>Query Point</a>
+        <a href='/bench/geojson-rendering' class='button'>GeoJSON Rendering</a>
+    </div>
```

Comment:
Generating it from the [`benchmarks` object in `bench/index.js`](https://github.com/mapbox/mapbox-gl-js/pull/2327/files#diff-4c5f2be87a037cceed511291a012024aL16) would be good.